src/main/kotlin/racingcar/domain/collection/Cars.kt

```diff
@@ -9,7 +11,28 @@ class Cars(
         cars.forEach(Car::run)
     }
 
-    fun getPositions(): List<Int> {
-        return cars.map { it.position }
+    fun getCarRecords(): List<Record> {
+        val records = mutableListOf<Record>()
+
+        getPositions().forEach { position ->
+            records.add(Record(position.first, position.second))
+        }
+
+        return records
+    }
+
+    private fun getPositions(): List<Pair<String, Int>> {
+        return cars.map { Pair(it.carName, it.position) }
+    }
```

Comment:
`List<Car>`를 `List<Pair<String, Int>>`로 변환하는 부분은 불필요해보이네요. 아래처럼 정리하면 어떨까요?

```suggestion
    fun getCarRecords(): List<Record> {
        return cars.map { Record(it.carName, it.position) }
    }
```

Reply:
해당 [커밋](https://github.com/next-step/kotlin-racingcar/pull/632/commits/f151d271b06efe8c18880620245472d95cb11b5b)에 반영하였습니다. ~